source/base/Event.ooc

```diff
@@ -17,14 +17,20 @@
 
 Event: class {
 	_head: Func
-	head ::= this _head
 	_tail: This
+	head ::= this _head
```

Comment:
Hmm I'm not sure about this. I think it could be good to have the corresponding property right next to its member. Will need to think about this.